metatron-app/metatron-discovery
# fn-fix_metadata_detail_error

### Description

Change Date format
change modal's information from updated time to created time
**Related Issue** : 
METATRON-3019



### How Has This Been Tested?

1. Management -> metadata -> click random list item
2. check time's format
3. click more button and check created time is showing
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
